commands/cli/parse.go

```diff
@@ -429,7 +439,32 @@ func appendFile(fpath string, argDef *cmds.Argument, recursive, hidden bool) (fi
 		}
 	}
 
-	return files.NewSerialFile(path.Base(fpath), fpath, hidden, stat)
+	// special cases for Windows drive roots i.e. X:\ and their long form \\?\X:\
+	// drive path must be preserved and not converted to `X:`, `X:.`, or `/`
+	switch len(fpath) {
+	case 3:
+		// X: will be cleaned to `X:.` which is not what is intended, a case for handling dot relative paths is above
+		if fpath[0] != '/' && fpath[1:3] == ":." {
```

Comment:
Hrm, we need to special case this to only run on windows. In linux, `X:.` is a perfectly valid filename